util/randomutil/randomutil.go

```diff
@@ -0,0 +1,17 @@
+package randomutil
+
+import (
+	"math/rand"
+	"time"
+)
+
+type RandomGenerator interface {
+	GenerateInt63() int64
+}
+
+type RandomNumberGenerator struct{}
+
+func (RandomNumberGenerator) GenerateInt63() int64 {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
This is not necessary since we initialize the random seed in `main.go`. There is also no need to reinitialize the random seed every time we need to generate a random number. This can just be an interface wrapper for rand.Int63().

Reply:
done in https://github.com/prebid/prebid-server/pull/2679/commits/868dbdbbdace65b645bb5e28ccb5df68aadbfa9b